breaks.py

```diff
@@ -1 +1,127 @@
+
+    heading = Label(root, text = "Notifications", font = ( "", int(calculateFontSize(WIDTH, hu*20) * 1.5)), fg = "black", bg = "#bdbdbd")
+    heading.place(x = 0, y = 0, width = WIDTH, height = hu*20)
+
+    productiveTimeLabel = Label(root, text = "Enter the amount of time(in minutes) between breaks", font = ( "", int(calculateFontSize(wu*70, hu*15) )), fg = "white", bg = "black", justify = LEFT)
+    productiveTimeLabel.place(x = 5*wu, y = 25*hu, height = hu*10)
+
+    productiveTimeEntry = Entry(root, font = ("", int(calculateFontSize(wu * 90, hu*10)*2/3)), highlightbackground="red")
+    productiveTimeEntry.place(x = 5*wu, y = 35*hu, width = wu*90, height = hu*10)
+
+    breakTimeLabel = Label(root, text = "Enter the length(in minutes) of breaks", font = ( "", int(calculateFontSize(wu*70, hu*15) )), fg = "white", bg = "black", justify = LEFT)
+    breakTimeLabel.place(x = 5*wu, y = 45*hu, height = hu*10)
+
+    breakTimeEntry = Entry(root, font = ("", int(calculateFontSize(wu * 90, hu*10)*2/3)), highlightbackground="red")
+    breakTimeEntry.place(x = 5*wu, y = 55*hu, width = wu*90, height = hu*10)
+
+    currentStatusLabel = Label(root, text = "Current Status : N/A", font = ( "", int(calculateFontSize(wu*40, hu*10) )), fg = "white", bg = "black", justify = LEFT)
+    currentStatusLabel.place(x = 5*wu, y = 65*hu, height = hu*10)
+
+    helpButton = standardButton(root, "Not Working? Click For Help", lambda : showHelp(root, WIDTH, HEIGHT), wu*50, hu*8)
+    helpButton.place(x = wu*55, width = wu*40, height = hu*8, y = 65*hu)
+
+    startButton = RoundedButton(root, "black", 25*wu, 25*hu, "Start", int(calculateFontSize(25*wu, 25*hu)*1.2), startNotifications, "#00ff6e")
+    startButton.place(x = 5*wu, y = 75*hu, width = 25*wu, height = 25*hu)
+    
+    backButton = RoundedButton(root, "black", 30*wu, 25*hu, "Go Back", int(calculateFontSize(30*wu, 25*hu)*1.2), root.place_forget, "#00bbff")
+    backButton.place(x = 35*wu, y = 75*hu, width = 30*wu, height = 25*hu)
+
+    stopButton = RoundedButton(root, "black", 25*wu, 25*hu, "Stop", int(calculateFontSize(25*wu, 25*hu)*1.2), stopNotifications, "#00ff6e")
+    stopButton.place(x = 70*wu, y = 75*hu, width = 25*wu, height = 25*hu)
+
+    root.mainloop()
+
+def showNotifications(title, description):
+    showNotification = threading.Thread(target = lambda : toast.show_toast(title, description))
+    showNotification.start()
+    return showNotification
+
+def startNotifications():
+
+    showNotifications("Notifications On", "You will recieve notifications like this for taking breaks")
+
+    userNotificationData = {
+        "notifications":{"showNotifications":True}
+    }
+
+    modifyUserActivity(userNotificationData)
+
+    def checkToLeave(lengthOfBreak):
+        start = time.time()
+        if lengthOfBreak == "":
+            messagebox.showinfo("Error", "Enter a valid time")
+            return True
+
+        while time.time() - start < float(lengthOfBreak) * 60:
```

Comment:
We are sleeping for each second and then continuing in order to keep checking if the program has to be stopped. Else if we just do time.sleep(lengthOfBreak*60) then we would have to wait until that is over in order to check if we need to leave.